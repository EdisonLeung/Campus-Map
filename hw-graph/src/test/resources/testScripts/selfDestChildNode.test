# Creates 2 nodes and tests having one edge reference the node it is coming from and the
# other edge point from that node to the other node

# Create graph
CreateGraph selfDestChildNode

# Add 2 nodes
AddNode selfDestChildNode a
AddNode selfDestChildNode b

# Add one edge that references the node it comes from
AddEdge selfDestChildNode a a AA

# Add second edge that references the other node
AddEdge selfDestChildNode a b AB

# Print the nodes and the children
ListNodes selfDestChildNode
ListChildren selfDestChildNode a
ListChildren selfDestChildNode b