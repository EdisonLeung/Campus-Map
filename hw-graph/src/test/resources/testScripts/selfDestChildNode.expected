# Creates 2 nodes and tests having one edge reference the node it is coming from and the
# other edge point from that node to the other node

# Create graph
created graph selfDestChildNode

# Add 2 nodes
added node a to selfDestChildNode
added node b to selfDestChildNode

# Add one edge that references the node it comes from
added edge AA from a to a in selfDestChildNode

# Add second edge that references the other node
added edge AB from a to b in selfDestChildNode

# Print the nodes and the children
selfDestChildNode contains: a b
the children of a in selfDestChildNode are: a(AA) b(AB)
the children of b in selfDestChildNode are: